services/velero/3.4.0/velero.yaml

```diff
@@ -13,7 +13,7 @@ spec:
   wait: true
   interval: 6h
   retryInterval: 1m
-  path: ./services/velero/3.3.1/helmrelease
```

Comment:
there is also a pre-install kustomization that i don't see in this PR, maybe it got removed with conflicts or something

Reply:
oh shoot! lemme fix that..

Reply:
https://github.com/mesosphere/kommander-applications/pull/947/commits/bf567e6ba87ac9686425fb0be864558fb8e895a3 think this'll fix it